plugin/src/sbt-test/sbt-mu-srcgen/compendium/build.sbt

```diff
@@ -1,5 +1,10 @@
 version := sys.props("version")
 
-libraryDependencies ++= Seq(
-  "io.higherkindness" %% "mu-rpc-server" % sys.props("mu")
-)
+lazy val root = project
+  .in(file("."))
+  .enablePlugins(SrcGenPlugin)
+  .settings(Seq(
+    libraryDependencies ++= Seq(
+  "io.higherkindness" %% "mu-rpc-server" % sys.props("mu"),
```

Comment:
Same here and the rest of the scripted tests

Reply:
yeah I'll make that change across the board